app/controllers/sign_up/completions_controller.rb

```diff
@@ -25,7 +25,7 @@ def update
       if decider.go_back_to_mobile_app?
         sign_user_out_and_instruct_to_go_back_to_mobile_app
       else
-        redirect_to sp_session_request_url_without_prompt_login
+        redirect_to sp_session_request_url_without_prompt_login || account_url
```

Comment:
WDYT if adding parens so that the precedence is clearer?

```suggestion
        redirect_to(sp_session_request_url_without_prompt_login || account_url)
```

Reply:
i like!